src/test/java/seedu/planitarium/command/AddPersonTest.java

```diff
@@ -1,4 +1,59 @@
 package seedu.planitarium.command;
 
+import org.junit.jupiter.api.Test;
+import seedu.planitarium.commands.AddPersonCommand;
+import seedu.planitarium.exceptions.PlanITariumException;
+import seedu.planitarium.family.Family;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.fail;
+
 public class AddPersonTest {
+    Family family = new Family();
+    AddPersonCommand add;
+    protected static final String NAME_ERROR_MSG =
+            "Unknown error is detected from 'Empty string after `/n`', please check again.";
+    protected static final String EMPTY_GROUPINX_ERROR_MSG =
+            "Unknown error is detected from 'Empty string after `/g`', please check again.";
+    protected static final String INVALID_GRPINX_ERROR_MSG =
+            "Unknown error is detected from 'Invalid group index `4`', please check again.";
+
+    @Test
+    void addPerson_invalidName_fail() {
+        try {
+            add = new AddPersonCommand(CommandsForTesting.ADDPERSON4, family);
```

Comment:
Would it be better to include a `fail()` statement right after this command in the `try` block as well? Then you could catch those that did not even throw errors.